jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/test/util/AbstractKieServicesBaseTest.java

```diff
@@ -110,4 +115,33 @@ protected KieFileSystem createKieFileSystemWithKProject(KieServices ks) {
         return kfs;
     }
 
+    protected void assertTask(UserTaskInstanceDesc userTask, long processInstanceId) {
+        assertTask(userTask, Long.toString(processInstanceId));
+    }
```

Comment:
Why do we need this intermediate step and not do the conversion inside the assert method below? As this method doesn't do anything more.

Reply:
This method is a bridge between processInstanceId and correlationKey. In most test, correlationKey is equal to processInstanceId. This is doing the briding. If this method is removed Long.toString will have to be invoked in a lot of test.

Reply:
I see now, thanks.